Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19.10-16 as builder
+FROM registry.redhat.io/ubi8/go-toolset:1.19 as builder
```

Comment:
Should `egistry.access.redhat.com` be kept?

Reply:
Thanks @jbusche, I agree with you.

Note these are pointing to the exact same image. It's only the access that changes.

Reply:
Actually CI is failing for that exact reason.

Reply:
Thanks guys, I wan't aware. I'll update now